bin/diagnostics_functions.js

```diff
@@ -0,0 +1,63 @@
+const process = require('process');
+const util = require('./shared_utils');
+const config = require('./config').config;
+const request = require('superagent');
+const fs = require('fs');
+const resolve = require('path').resolve;
+
+module.exports.dispatcher = function (args) {
+    if (args.length === 0) {
+        printModuleHelp();
+        process.exit(1);
+    }
+    switch (args[0].toLowerCase()) {
+        case 'run': {
```

Comment:
should we print help if the either output folder or fileName is missing in the command

Reply:
Hmm, I don't think so. Both are optional and there is a default.